cli.js

```diff
@@ -14,6 +14,7 @@ const cli = meow(`
 
 	Options
 	  -f, --force  Force kill
+	  -v, --verbose Include arguments in process search
```

Comment:
Should be two spaces between the flag and description, and you need to align the above flag.